src/tests/test_gaussian_quadratures.cc

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright 2009-2020 The VOTCA Development Team (http://www.votca.org)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+#define BOOST_TEST_MAIN
+
+#define BOOST_TEST_MODULE gaussian_quadratures_test
+
+// Standard includes
+#include <fstream>
+
+// Third party includes
+#include <boost/test/unit_test.hpp>
+
+// Local VOTCA includes
+#include "votca/xtp/quadrature_factory.h"
+
+// VOTCA includes
+#include <votca/tools/eigenio_matrixmarket.h>
+
+using namespace votca::xtp;
+using namespace votca;
+
+// defines a Gaussian as integration test
+// result should by ~sqrt(pi)
+class FunctionEvaluation {
+ public:
+  FunctionEvaluation(){};
+
+  double operator()(Index j, double point, bool symmetry) const {
+    double factor;
+    if (symmetry) {
+      factor = 2.0;
+    } else {
+      factor = 1.0;
+    }
+
+    return factor * exp(-std::pow(point, 2));
+  }
+};
+
```

Comment:
modified gauss-legendre is missing from the tests

Reply:
added